src/core/blocks/validate_block.c

```diff
@@ -13,10 +13,45 @@
 #include "create_block.h"
 #include "mempool.h"
 #include "blockchain.h"
-#include "utxo_pool.h"
-#include "wallet_pool.h"
+#include "utxo_to_tx.h"
+#include "double_spend_set.h"
 #include "validate_tx.h"
 
```

Comment:
Sorry, I reordered these functions without thinking of the diff it would make. I don't think any of them changed.